pcsx2/gui/RecentIsoList.cpp

```diff
@@ -178,6 +178,23 @@ void RecentIsoManager::InsertIntoMenu( int id )
 
 	if ( !exists )
 		curitem.ItemPtr->Enable( false );
+	else
+		curitem.ItemPtr->Enable( !g_Conf->AskOnBoot );
+}
+
+void RecentIsoManager::EnableItems(bool display)
+{
+	if (m_Items.empty())
+		return;
```

Comment:
The early return isn't really necessary in this case - the for loop will just be skipped if `m_Items` is empty.